core/src/main/java/org/apache/calcite/rel/rules/AggregateValuesRule.java

```diff
@@ -53,6 +56,20 @@
  * Any non-empty {@code GROUP BY} clause will return one row per group key
  * value, and each group will consist of at least one row.
  *
+ * <p>This is useful because such as {@link SubQueryRemoveRule}
+ * doesn't distinct values for IN filter values.
+ *
+ * <blockquote><code>
+ * LogicalAggregate(group=[{0}])
+ *   LogicalValues(tuples=[[{ 1 }, { 1 }, { 3 }]])
+ * </code></blockquote>
+ *
+ * <p>should be converted to:
+ *
+ * <blockquote><code>
```

Comment:
What happened to the aggregation here? Is it now redundant and removed due to the deduplication? If so, please state it explicitly in the comment.

Reply:
@asolimando Thanks, I had changed it to: 
```
With the Rule would deduplicate the values and convert to:
```